app/decorators/staff/time_slot_decorator.rb

```diff
@@ -64,11 +64,7 @@ def session_confirmation_notes
   end
 
   def display_title
-    if object.program_session.present?
-      object.program_session.title
-    else
-      object.title
-    end
+    object.program_session.try(:title) || object.title
```

Comment:
That makes sense. I think for similar logic with user & speaker names & emails, the model was in charge of this.